source/integrations/jira.rst

```diff
@@ -72,6 +79,23 @@ This is because `Site URL <https://docs.mattermost.com/administration/config-set
 
 To resolve it, set your Site URL in **System Console > General > Configuration**.
 
+Why doesn't my JIRA webhook post any messages?
+.....................................................
+
+Try the following troubleshooting steps:
+
+1. Confirm ``user`` field is set in **System Console > JIRA (Beta)**. The integration needs to be attached to a user account for the webhook to post messages.
+2. Confirm the team URL and channel URL you specified in the JIRA webhook URL is in lower case.
+3. For issue updated events, only status changes when the ticket is reopened, or when resolved/closed are supported. If you'd like to see support for additional events, `let us know <https://mattermost.uservoice.com/forums/306457-general>`_.
+4. If you specified a JQL query in your JIRA webhook page, paste the JQL to JIRA issue search and make sure it returns results. If it doesn't, the query may be incorrect. Refer to the `Atlassian documentation <https://confluence.atlassian.com/jirasoftwarecloud/advanced-searching-764478330.html>`_ for help.
+
+If you are still having trouble with configuration, feel free to post in our `Troubleshooting forum <https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150>`_ and we'll be happy to help with issues during setup.
+
+How do I disable the plugin quickly in an emergency?
+.....................................................
+
+You can disable the JIRA integration any time from **System Console > JIRA (Beta)**. Requests will stop immediately with an error code in **System Console > Logs**. No posts are created until the integration is re-enabled.
```

Comment:
I would make it clear that you just need to flip it to false, otherwise people might think there is a disable button